### PR TITLE
Cleaned up logging

### PR DESCRIPTION
**Pull request by rain-on**

### CODE REVIEW

File: acceptance-tests/src/test/resources/log4j2.xml [hunk 1]
```diff
@@ -2,13 +2,30 @@
...
   <Loggers>
+    <Logger name="tech.pegasys.web3signer.SubProcessLog" level="INFO" additivity="false">
+      <AppenderRef ref="SubProcessConsole" />
+    </Logger>
+    <Logger name="com.zaxxer.hikari" level="${dependency.log.level}" additivity="false"/>
+    <Logger name="org.flywaydb.core" level="${dependency.log.level}" additivity="false"/>
+    <Logger name="org.apache.http" level="${dependency.log.level}" additivity="false"/>
+    <Logger name="io.restassured.internal" level="${dependency.log.level}" additivity="false"/>
+    <Logger name="io.netty" level="${dependency.log.level}" additivity="false"/>
+    <Logger name="io.swagger.v3.parser.OpenAPIV3Parser" level="${dependency.log.level}" additivity="false"/>
```
Comment (jframe) [inline acceptance-tests/src/test/resources/log4j2.xml:25]: Do we just want set all swagger logs to warn? i.e. io.swagger as the name. And similarly for com.atlassian.oai?

File: core/src/main/java/tech/pegasys/web3signer/core/service/http/handlers/signing/Eth2SignForIdentifierHandler.java [hunk 2]
```diff
@@ -93,7 +94,7 @@ public void handle(final RoutingContext routingContext) {
                 }
               },
               () -> {
-                LOG.trace("Identifier not found {}", identifier);
+                LOG.warn("Identifier not found {}", identifier);
```
Comment (jframe) [inline core/src/main/java/tech/pegasys/web3signer/core/service/http/handlers/signing/Eth2SignForIdentifierHandler.java:97]: Don't think we need this log at all as when no signer was found we log an error in DefaultArtifactSignerProvider, log at warn here and then finally log in the LogErrorHandler so all up we get 3 logs entries.

e.g.
```
2020-10-06 15:22:55.650+10:00 | vert.x-worker-thread-5 | ERROR | DefaultArtifactSignerProvider | No signer was loaded matching identifier '0x900884a719c418272c1c2b038f3dd5629a4bf2465d30963d181f454565f4236c3441aa49049c9fedac57c381fb067a2a'
2020-10-06 15:22:55.650+10:00 | vert.x-worker-thread-5 | WARN  | Eth2SignForIdentifierHandler | Identifier not found 0x900884a719c418272c1c2b038f3dd5629a4bf2465d30963d181f454565f4236c3441aa49049c9fedac57c381fb067a2a
2020-10-06 15:22:55.650+10:00 | vert.x-eventloop-thread-4 | ERROR | LogErrorHandler | Failed request: http://localhost:9000/api/v1/eth2/sign/0x900884a719c418272c1c2b038f3dd5629a4bf2465d30963d181f454565f4236c3441aa49049c9fedac57c381fb067a2a
```

File: acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/runner/Web3SignerProcessRunner.java [hunk 3]
```diff
@@ -115,7 +115,7 @@ private void printOutput() {
         new BufferedReader(new InputStreamReader(process.getInputStream(), UTF_8))) {
       String line = in.readLine();
       while (line != null) {
-        PROCESS_LOG.info("Web3SignerProc: {}", line);
+        PROCESS_LOG.info(line);
```
Comment (jframe) [inline acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/runner/Web3SignerProcessRunner.java:118]: Is this confusing without the prefix? How do we know which log lines are from web3signer?

File: acceptance-tests/src/test/resources/log4j2.xml (hunk 1 above)
Comment (usmansaleem) [reply acceptance-tests/src/test/resources/log4j2.xml:25]: 👍

File: acceptance-tests/src/test/resources/log4j2.xml [hunk 4]
```diff
@@ -2,13 +2,30 @@
...
     <Property name="root.log.level">DEBUG</Property>
+    <Property name="dependency.log.level">WARN</Property>
   </Properties>
 
   <Appenders>
     <Console name="Console" target="SYSTEM_OUT">
-      <PatternLayout pattern="%d{yyyy-MM-dd HH:mm:ss.SSSZZZ} | %t | %-5level | %c{1} | %msg%n" />    </Console>
+      <PatternLayout pattern="%d{yyyy-MM-dd HH:mm:ss.SSSZZZ} | %t | %-5level | %c{1} | %msg%n" />
+    </Console>
+    <Console name="SubProcessConsole" tareget="SYSTEM_OUT">
```
Comment (usmansaleem) [inline acceptance-tests/src/test/resources/log4j2.xml:12]: mispell .... `target`

**Comment by usmansaleem** [comment]
The log4j files in test resources can be appended with `-test` i.e. `log4j2-test.xml`. This allows these files to take precedence over standard files if they are on classpath.

File: acceptance-tests/src/test/resources/log4j2.xml (hunk 1 above)
Comment (usmansaleem) [reply acceptance-tests/src/test/resources/log4j2.xml:25]: I think all of the "dependent" libraries logs should be warn ... unless we want to debug those libraries themselves.

File: core/src/main/java/tech/pegasys/web3signer/core/service/http/handlers/signing/Eth2SignForIdentifierHandler.java (hunk 2 above)
Comment (usmansaleem) [reply core/src/main/java/tech/pegasys/web3signer/core/service/http/handlers/signing/Eth2SignForIdentifierHandler.java:97]: 👍

File: acceptance-tests/src/test/resources/log4j2.xml (hunk 1 above)
Comment (rain-on) [reply acceptance-tests/src/test/resources/log4j2.xml:25]: yeah, that's fine.

File: core/src/main/java/tech/pegasys/web3signer/core/service/http/handlers/signing/Eth2SignForIdentifierHandler.java (hunk 2 above)
Comment (rain-on) [reply core/src/main/java/tech/pegasys/web3signer/core/service/http/handlers/signing/Eth2SignForIdentifierHandler.java:97]: done

File: acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/runner/Web3SignerProcessRunner.java (hunk 3 above)
Comment (rain-on) [reply acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/runner/Web3SignerProcessRunner.java:118]: They still have "subprocess" as the first entry in the line due to the name of the logger.

File: acceptance-tests/src/test/resources/log4j2.xml (hunk 4 above)
Comment (rain-on) [reply acceptance-tests/src/test/resources/log4j2.xml:12]: good catch :)

**Review by usmansaleem — APPROVED** [review]
minor comments, looks good otherwise.

**Review by usmansaleem — APPROVED** [review]
LGTM

**Review by jframe — APPROVED** [review]

**Pull request merged**